test/box/fix-memory-corruption.test.lua

```diff
@@ -0,0 +1,23 @@
+env = require('test_run')
+fiber = require('fiber')
+net_box = require('net.box')
```

Comment:
I can't believe, that  you didn't know I will ask - where is a comment explaining the bug? And if you knew, why didn't you add it?

Reply:
fixed